.taprc

```diff
@@ -2,5 +2,5 @@ esm: false
 ts: false
 jsx: false
 flow: false
-timeout: 500
+timeout: 900
```

Comment:
May be this is not ideal, but in [some tests](https://github.com/mcollina/autocannon/blob/master/test/httpClient.test.js#L580) we're waiting for a couple of seconds which may be messing up with tap timeouts on windows.

Reply:
Fails after 3 mins even though the timeout is 15 mins now. I'll investigate more.

Reply:
Splitting some tests into multiple blocks in test/runRate.test.js seems to have fixed the issue.
_Or may be it's appearing as so._ 😅